<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Animation3DWidget</class>
 <widget class="QWidget" name="Animation3DWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1301</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Keyframe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboKeyframe"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnAddKeyframe">
       <property name="toolTip">
        <string>Add keyframe</string>
       </property>
       <property name="text">
        <string> + </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnRemoveKeyframe">
       <property name="toolTip">
        <string>Remove keyframe</string>
       </property>
       <property name="text">
        <string> - </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnEditKeyframe">
       <property name="toolTip">
        <string>Edit keyframe</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnDuplicateKeyframe">
       <property name="toolTip">
        <string>Duplicate keyframe</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="btnPlayPause">
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderTime">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBothSides</enum>
       </property>
       <property name="tickInterval">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cboKeyframe</tabstop>
  <tabstop>btnAddKeyframe</tabstop>
  <tabstop>btnRemoveKeyframe</tabstop>
  <tabstop>btnEditKeyframe</tabstop>
  <tabstop>btnDuplicateKeyframe</tabstop>
  <tabstop>btnPlayPause</tabstop>
  <tabstop>sliderTime</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
